### DISCUSSION TITLE
how can I get it working in wsl2?

### DISCUSSION THREAD

**Discussion by simonm3** [quarto-dev/quarto-cli#3962] [Q&A] [answered]
I installed quarto in wsl2 and the vs code extension. The quarto commands appear in the command palette (create project, new document etc..). However when I select any of the commands nothing happens.

What have I missed?

**Comment by simonm3** [comment, marked as answer]
Managed to get working. I think it was a folder permissions issue or broken link.

**Reply by Nurnberg-Meistersinger** [reply]
All is done, I've installed it into my WSL correctly. Here is the list of commands how to get it done.

- wget https://github.com/quarto-dev/quarto

- cli/releases/download/v1.3.450/quarto-1.3.450-linux-amd64.deb

- sudo dpkg -i quarto-1.3.450-linux-amd64.deb

- quarto --version

- rm quarto-1.3.450-linux-amd64.deb

**Reply by JacobBumgarner** [reply]
Also Posit's documentation for installing quarto using `.deb`:
https://docs.posit.co/resources/install-quarto/#download-and-install-quarto-using-deb-file

**Comment by simonm3** [comment]
No idea how it started working. However I would make sure all data is
inside wsl not a link to windows drive. Once that works then you can try
moving the files to windows.

Pretty sure it was about permissions especially on files created on windows

**Reply by mcanouil** [reply]
Great you managed to get it to work.
Note that Quarto (Deno) has a known issue with network drives, thus we highly recommend to avoid working on those when possible.
